tests/integration/test_consultation_page.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from django.contrib.auth.models import Group
+
+from consultation_analyser.constants import DASHBOARD_ACCESS
+from consultation_analyser.consultations.dummy_data import create_dummy_consultation_from_yaml
+from consultation_analyser.consultations.models import Question
+from consultation_analyser.factories import UserFactory
+from tests.helpers import sign_in
+
+
+@pytest.mark.django_db
+def test_consultation_page(django_app):
+    user = UserFactory()
```

Comment:
could/should this be a fixture to ensure that it gets torn down after the test? 

no need to actually implement this... more of an open question